src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -361,37 +340,35 @@ const DOCS: {[key: string]: DocCategory[]} = {
           id: 'bottom-sheet',
           name: 'Bottom Sheet',
           summary: 'A large interactive panel primarily for mobile devices.',
-          examples: ['bottom-sheet-overview']
+          exampleSpecs: {
+            prefix: 'bottom-sheet',
```

Comment:
Curious why some prefixes end with a `-` but others don't. Is this intended?

Reply:
Oops it was a mistake. Now they all end with `-`